src/wgxpath.js

```diff
@@ -221,16 +222,17 @@ wgxpath.XPathNSResolver_ = function(node) {
 
 
 /**
- * Installs the library. This is a noop if native XPath is available.
+ * Installs the library. This is a noop if native XPath is available or the 2nd parameter is true.
```

Comment:
Should it be "This is a noop if native XPath is available and the 2nd parameter is false."?